opentech/apply/users/templates/wagtailusers/users/create.html

```diff
@@ -0,0 +1,26 @@
+{% extends "wagtailusers/users/create.html" %}
+
+{% block fields %}
+    {% if form.separate_username_field %}
+        {% include "wagtailadmin/shared/field_as_li.html" with field=form.username_field %}
+    {% endif %}
+    {% include "wagtailadmin/shared/field_as_li.html" with field=form.email %}
+    {% include "wagtailadmin/shared/field_as_li.html" with field=form.full_name %}
+
+    {% comment %}
+        First/last name hidden input with dummy values because.. Wagtail admin
+        See opentech.apply.users.forms.CustomUserCreationForm
+    {% endcomment %}
+    {{ form.first_name }}
```

Comment:
Does `{% include "wagtailadmin/shared/field_as_li.html" with field=form.first_name %}` render something?

Using  `extra_fields` block would be preferable to copying everything in.

https://github.com/wagtail/wagtail/blob/76975525f7a3757b280572165f8717cc4c13bba6/wagtail/users/templates/wagtailusers/users/create.html#L27

Reply:
Yes. The field label. 
![screen shot 2018-02-28 at 14 26 01](https://user-images.githubusercontent.com/31622/36792854-a8f22032-1c93-11e8-982f-44b2eeb97d6c.png)

Reply:
Could be pulled into a shared include, but not too fussed at this point now.